cmd/bankapp/main.go

```diff
@@ -25,19 +24,18 @@ func main() {
 	s := r.PathPrefix("/v1/users").Subrouter()
 
 	userSessionDuration := time.Second * 300
-	mongoSessionStore := persistence.NewMongoSessionStore(*session, "bank")
 
-	userStore := domain.UserStore(mongoSessionStore)
-	userSessionStore := domain.SessionStore(mongoSessionStore)
-	accountStore := domain.AccountStore(mongoSessionStore)
+	sessionStore := persistence.NewSessionStore(*session, "bank")
```

Comment:
It would be better point to *mgo.Database to be passed instead of name of the DB as it is duplicated.